app/controllers/config_controller.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+# ConfigController shows the main page.
+class ConfigController < ApplicationController
+  def index
```

Comment:
If they are blank, you can skip writing them at all, same with the others. If you only add the view it should work.